### DISCUSSION TITLE
Any way to limit log error notifications?

### DISCUSSION THREAD

**Discussion by Spectrum-Prez** [Rem0o/FanControl.Releases#1890] [General]
FanControl is giving me an error message every hour or so that has something to do with the water flow/temp meter I have installed in my cooling loop. I obviously don't expect every bit of hardware and software to be compatible or to work perfectly together without a Plugin, but these stacking error messages can be a bit annoying to have to click away one at a time whenever I open the main FanControl window. Is there any way to not have these errors pop up, or to suppress errors specific to this one sensor?

**Comment by Rem0o** [comment]
Is said error coming from the HWInfo plugin?

**Comment by Spectrum-Prez** [comment]
I don't have the HWInfo plugin installed, so I doubt it. The full error message is below, although I honestly was just hoping there would be a way to suppress the error notifications. 

> 7/4/2023 11:15:59 PM: Unhandled exception in FanControl v162.0.0.0
> 7/4/2023 11:15:59 PM: System.IO.IOException: GetFeature failed. ---> System.ComponentModel.Win32Exception: The operation completed successfully
>    --- End of inner exception stack trace ---
>    at HidSharp.Platform.Windows.WinHidStream.GetFeature(Byte[] buffer, Int32 offset, Int32 count)
>    at LibreHardwareMonitor.Hardware.Controller.AquaComputer.MPS.Update() in C:\Users\Remi\source\repos\FanControl\LibreHardwareMonitor\LibreHardwareMonitorLib\Hardware\Controller\AquaComputer\MPS.cs:line 82
>    at FanControl.Domain.BackendProviders.LHM.UpdateVisitor.VisitHardware(IHardware hardware)
>    at LibreHardwareMonitor.Hardware.Computer.Traverse(IVisitor visitor) in C:\Users\Remi\source\repos\FanControl\LibreHardwareMonitor\LibreHardwareMonitorLib\Hardware\Computer.cs:line 397
>    at FanControl.Domain.BackendProviders.LHM.LHMBackendProvider.Update()
>    at System.Collections.Generic.List`1.ForEach(Action`1 action)
>    at FanControl.Domain.ComputerAccessLayer.Update()
>    at FanControl.Domain.ApplicationClock.DoActions()
> --- End of stack trace from previous location where exception was thrown ---
>    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
>    at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
>    at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
> 7/4/2023 11:15:59 PM: System.ComponentModel.Win32Exception (0x80004005): The operation completed successfully

**Reply by Rem0o** [reply]
Suppressing errors arbitrarily isn't the way to go.  If an important error pop up, well, you gotta see it. In this case, it's a bad error management in the Aquacomputer MPS stuff. 

Made a PR to fix it: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/pull/1113

**Reply by Spectrum-Prez** [reply]
Thank you for that, and for monitoring these discussions so closely.
